src/common/utxobased/db/makeProcessor.ts

```diff
@@ -260,6 +260,7 @@ export async function makeProcessor(
         // Return all UTXOs if no UTXO ids are specified
         if (utxoIds.length === 0) {
           const { data } = await tables.utxoById.dumpData('')
+          if (data[''] == null) return []
```

Comment:
You can do this instead: `return Object.values(data[''] ?? {})`

Reply:
Yes. Nice suggestion!